docs/guides/js-waku/index.md

```diff
@@ -30,26 +30,64 @@ yarn add @waku/sdk
 </TabItem>
 </Tabs>
 
-You can also use the `js-waku` SDK via a CDN without installing it on your system:
+You can also use the `@waku/sdk` package via a CDN without installing it on your system:
 
 ```js
 import * as waku from "https://unpkg.com/@waku/sdk@latest/bundle/index.js";
 ```
 
+## Message Structure
+
+We recommend creating a message structure for your application using [Protocol Buffers](https://protobuf.dev/) for the following reasons:
+
+1. **Consistency:** Ensures uniform message format for easy parsing and processing.
```

Comment:
Another criteria is payload size. protobuf allows minimum payload overhead, especially when handing byte arrays, Which is much better than using JSON for example where it has a lot of overhead, especially by using utf8 to transport a byte array